app/views/admin/items/index.html.erb

```diff
@@ -27,9 +27,9 @@
             </td>
             <td>
               <% if item.retired? %>
-                <%= link_to "Reactivate Accessory", admin_bike_shop_index_path(item, status: "retired"), method: "patch", data: {confirm: "Are you sure you want to reactivate #{item.title}?"} %>
-              <% else %>
-                <%= link_to "Retire Accessory", admin_bike_shop_index_path(item, status: "active"), method: "patch", data: {confirm: "Are you sure you want to retire #{item.title}?"} %>
+                <%= link_to "Reactivate Accessory", admin_bike_shop_path(item, item: {id: item.id, title: item.title, description: item.description, price: item.price, image_file_name: "img_default.jpg", image_content_type: "image/png", image_file_size: 359655, image_updated_at: "Mon, 16 Apr 2018 19:23:30 UTC +00:00"}, status: "retired"), method: "patch", data: {confirm: "Are you sure you want to reactivate #{item.title}?"} %>
```

Comment:
@anon0mys @jrambold @jamesrnelson For some reason, the item was not being passed in the parameters to the controller(just the item_id was being passed), so I did this janky thing to make it work.  If you figure it out, I'll give you a cookie.